packages/eas-cli/src/build/runBuildAndSubmit.ts

```diff
@@ -130,7 +130,7 @@ export async function runBuildAndSubmitAsync(
     projectDir,
     nonInteractive: flags.nonInteractive,
   });
-  const easJsonAccessor = new EasJsonAccessor(projectDir);
+  const easJsonAccessor = new EasJsonAccessorWithCustomErrorMessages(projectDir);
```

Comment:
IMO it's worthwhile figuring out how to enforce using `EasJsonAccessorWithCustomErrorMessages` in eas-cli. So we use it consistently across the codebase. I think it should be possible with some eslint rule.

Reply:
Sounds like a good idea, I will try to look at it